rails_twirp.gemspec

```diff
@@ -13,10 +13,9 @@ Gem::Specification.new do |spec|
 
   spec.files = `git ls-files`.split("\n")
 
-  # Rails has shipped an incompatible change in ActiveView, that was reverted in later versions
-  # but at this time has not been released as a 7.x version
-  # @see https://github.com/rails/rails/pull/51023
-  spec.add_runtime_dependency "rails", ">= 6.1.3", " < 7.1"
-  spec.add_runtime_dependency "twirp", ">= 1.9", "< 1.11"
+  spec.add_runtime_dependency "rails", ">= 6.1.3", "!= 7.1"
+  spec.add_runtime_dependency "twirp", ">= 1.9"
```

Comment:
**suggestion:** I remember that `1.11` version had a breaking change for us. It might be that this is still a case? 

We can mark this version is incompatible.
```suggestion
  spec.add_runtime_dependency "twirp", ">= 1.9", "!= 1.11"
```

Reply:
O really? Good point. Updating twirp in this gem keeps the current version anyway (1.10).

Reply:
@svanhesteren seems like 1.11 is not available anymore. Seems like this version was yanked.

https://rubygems.org/gems/twirp